README.md

```diff
@@ -50,64 +48,29 @@ cd ManiSkill2 && pip install -e .
 
 ---
 
-GPU is required to enable rendering for ManiSkill2. The rigid-body environments, powered by SAPIEN, are ready to use after installation.
-
-Test your installation:
+A GPU with the Vulkan driver installed is required to enable rendering in ManiSkill2. The rigid-body environments, powered by SAPIEN, are ready to use after installation. Test your installation:
 
 ```bash
 # Run an episode (at most 200 steps) of "PickCube-v0" (a rigid-body environment) with random actions
+# Or specify an environment by "-e ${ENV_ID}"
 python -m mani_skill2.examples.demo_random_action
 ```
 
 Some environments require **downloading assets**. You can download all the assets by `python -m mani_skill2.utils.download_asset all` or download task-specific assets by `python -m mani_skill2.utils.download_asset ${ENV_ID}`.
+Please refer to our [documentation](https://haosulab.github.io/ManiSkill2/concepts/environments.html) for all supported environments, and which environments require downloading assets.
 
 ---
 
 The soft-body environments are based on SAPIEN and customized [NVIDIA Warp](https://github.com/NVIDIA/warp), which requires **CUDA toolkit >= 11.3 and gcc** to compile. Please refer to the [documentation](https://haosulab.github.io/ManiSkill2/getting_started/installation.html#warp-maniskill2-version) for more details about installing ManiSkill2 Warp.
 
-For soft-body environments, you need to make sure only 1 CUDA device is visible:
-
-``` bash
-# Select the first CUDA device. Change 0 to other integer for other device.
-export CUDA_VISIBLE_DEVICES=0
-```
-
-If multiple CUDA devices are visible, the environment will give an error. If you
-want to interactively visualize the environment, you need to assign the id of
-the GPU connected to your display (e.g., monitor screen).
+---
 
-All soft-body environments require runtime compilation and cache generation. You
-can run the following to compile and generate cache in advance. **This step is
-required before you run soft body environments in parallel with multiple processes.**
+Besides, we provide a docker image (`haosulab/mani-skill2`) on [Docker Hub](https://hub.docker.com/repository/docker/haosulab/mani-skill2/general) and its corresponding [Dockerfile](./docker/Dockerfile).
```

Comment:
grammar / word flow

We further provide a docker image (`haosulab/mani-skill2`) on the [Docker Hub](https://hub.docker.com/repository/docker/haosulab/mani-skill2/general) and its corresponding [Dockerfile](./docker/Dockerfile).